.github/MAPS_AND_AWAY_MISSIONS.md

```diff
@@ -1,13 +1,28 @@
 ## MAPS
 
-/tg/station currently comes equipped with five maps.
-
-* [BoxStation (default)](https://tgstation13.org/wiki/Boxstation)
+/tg/station currently has five maps in rotation.
 * [MetaStation](https://tgstation13.org/wiki/MetaStation)
 * [DeltaStation](https://tgstation13.org/wiki/DeltaStation)
+* [IceboxStation](https://tgstation13.org/wiki/IceboxStation)
+* [KiloStation](https://tgstation13.org/wiki/KiloStation)
+* [TramStation](https://tgstation13.org/wiki/Tramstation)
+
+Outdated, removed or not in rotation maps.
```

Comment:
This can now be "Outdated or removed maps." right?

Reply:
done